Build/SaveLoadGame.cc

```diff
@@ -160,7 +160,7 @@ static BYTE const* ExtractGameOptions(BYTE const* const data, GAME_OPTIONS& g)
 	EXTR_BOOL( d, g.fSciFi)
 	EXTR_U8(   d, g.ubDifficultyLevel)
 	EXTR_BOOL( d, g.fTurnTimeLimit)
-	EXTR_BOOL( d, g.fIronManMode)
+	EXTR_U8( d, g.ubGameSaveMode)
```

Comment:
Tested it on several saves, also ones I found in issues posted here. Everything worked correctly.
Since IronManMode was Boolean=TRUE before and saved as a "1", I don't see problems arising here either.